vmdb/app/helpers/container_project_helper/textual_summary.rb

```diff
@@ -40,4 +40,48 @@ def textual_ems
     end
     h
   end
+
+  def textual_container_routes
+    count_of_routes = @record.number_of(:container_routes)
+    label = ui_lookup(:tables => "container_routes")
```

Comment:
please extract common code in here as well.

``ruby
def textual_container_routes
  link_hash_of_counts(:container_routes, :container_route, "container_route_show_list"
end
```

You will need to write `link_hash_or_counts`. Let me know if the code is not simple.